servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/OptimizedHttp2FrameCodecBuilder.java

```diff
@@ -37,6 +37,10 @@ final class OptimizedHttp2FrameCodecBuilder extends Http2FrameCodecBuilder {
      */
     OptimizedHttp2FrameCodecBuilder(final boolean server) {
         this.server = server;
+        // We manage flushes at ST level and don't want netty to flush the preface & settings only. Instead, we write
+        // headers or entire message and flush them all together. Netty changed the default flush behavior starting from
+        // 4.1.78.Final. For context, see https://github.com/netty/netty/pull/12349.
+        flushPreface(false);
```

Comment:
For users which run older Netty with ServiceTalk I assume that this change will have no impact since the prior default was to not flush the preface, correct?

Reply:
Correct. You have a good point about older versions of netty. I've switched to `MethodHandle` to avoid breaking users who may depend on older netty.